### PR TITLE
Add jellyfin recipe (thanks @gpulido!)

### PR DESCRIPTION
**Pull request by gpulido**

### CODE REVIEW

File: manuscript/recipes/jellyfin.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Jellyfin
+
+[Jellyfin](https://jellyfin.org/) is best described as "_like [Emby](/recipes/emby) but really FOSS.
```
Comment (funkypenguin) [inline manuscript/recipes/jellyfin.md:3]: There's an unclosed quote and underscore here..

File: manuscript/recipes/jellyfin.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+1. [Docker swarm cluster](/ha-docker-swarm/design/) with [persistent shared storage](/ha-docker-swarm/shared-storage-ceph.md)
+2. [Traefik](/ha-docker-swarm/traefik) configured per design
+3. DNS entry for the hostname you intend to use, pointed to your [keepalived](ha-docker-swarm/keepalived/) IP
+
+## Preparation
+
+### Setup data locations
+
+We'll need a location to store Jellyfin's library data, config files, logs and temporary transcoding space, so create /var/data/jellyfin, and make sure it's owned by the user and group who also own your media data.
```
Comment (funkypenguin) [inline manuscript/recipes/jellyfin.md:19]: Use backticks for filesystem paths, i.e., `/var/data/jellyfin`

File: manuscript/recipes/jellyfin.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      - 8096:8096
+
+networks:
+  traefik_public:
+    external: true
+  internal:
+    driver: overlay
+    ipam:
+      config:
+        - subnet: 172.16.67.0/24
```
Comment (funkypenguin) [inline manuscript/recipes/jellyfin.md:79]: Add the subnet to https://github.com/geek-cookbook/geek-cookbook/blob/master/manuscript/reference/networks.md (and use the next available one)

File: manuscript/reference/networks.md [hunk 4]
```diff
@@ -53,4 +53,5 @@ In order to avoid IP addressing conflicts as we bring swarm networks up/down, we
 | [Harbor](https://geek-cookbook.funkypenguin.co.nz/recipes/graylog/)                                                   | 172.16.53.0/24 |
 | [Harbor-Clair](https://geek-cookbook.funkypenguin.co.nz/recipes/graylog/)                                             | 172.16.54.0/24 |
 | [Duplicati](https://geek-cookbook.funkypenguin.co.nz/recipes/duplicati/)                                              | 172.16.55.0/24 |
-| [Restic](https://geek-cookbook.funkypenguin.co.nz/recipes/restic/)                                                    | 172.16.56.0/24 |
\ No newline at end of file
+| [Restic](https://geek-cookbook.funkypenguin.co.nz/recipes/restic/)                                                    | 172.16.56.0/24 |
+| [Jellyfin](https://geek-cookbook.funkypenguin.co.nz/recipes/jellyfin/)                                                   | 172.16.57.0/24 |
```
Comment (SuperSandro2000) [inline manuscript/reference/networks.md:57]:
```suggestion
| [Jellyfin](https://geek-cookbook.funkypenguin.co.nz/recipes/jellyfin/)                                               | 172.16.57.0/24 |
```

Comment (SuperSandro2000) [inline manuscript/reference/networks.md:57]:
```suggestion
| [Restic](https://geek-cookbook.funkypenguin.co.nz/recipes/restic/)                                                    | 172.16.56.0/24 |
| [Jellyfin](https://geek-cookbook.funkypenguin.co.nz/recipes/jellyfin/)                                                | 172.16.57.0/24 |
```

File: manuscript/recipes/jellyfin.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ If it looks very similar as Emby, is because it started as a fork of it, but it
 
 ### Setup data locations
 
-We'll need a location to store Jellyfin's library data, config files, logs and temporary transcoding space, so create /var/data/jellyfin, and make sure it's owned by the user and group who also own your media data.
+We'll need a location to store Jellyfin's library data, config files, logs and temporary transcoding space, so create ```/var/data/jellyfin```, and make sure it's owned by the user and group who also own your media data.
```
Comment (SuperSandro2000) [inline manuscript/recipes/jellyfin.md:19]:
```suggestion
We'll need a location to store Jellyfin's library data, config files, logs and temporary transcoding space, so create ``/var/data/jellyfin``, and make sure it's owned by the user and group who also own your media data.
```

File: manuscript/recipes/jellyfin.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # Jellyfin
 
-[Jellyfin](https://jellyfin.org/) is best described as "_like [Emby](/recipes/emby) but really FOSS.
+[Jellyfin](https://jellyfin.org/) is best described as "_like [Emby](/recipes/emby) but really [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software#:~:text=Free%20and%20open%2Dsource%20software%20(FOSS)%20is%20software%20that,software%20and%20open%2Dsource%20software.)_".
```
Comment (SuperSandro2000) [inline manuscript/recipes/jellyfin.md:3]:
```suggestion
[Jellyfin](https://jellyfin.org/) is best described as "_like [Emby](/recipes/emby) but really [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software)_".
```
We don't need the chrome highlighting part.

**Pull request merged**